### ISSUE TITLE
Configure Adapter in Virtual Switch step fails, error details not usable

### ISSUE THREAD

**Issue by laurenbo** [Azure/AzureStackHCI-EvalGuide#23]
Hi, in WAC Cluster Creation, at step 2 "networking", at step 2.4 "virtual switch", I select the adapter named "VMs" on each node, click "Apply and Test", and I get "Couldn't configure adapter", on both adapters, with following error message: 
Exception calling "Invoke" with "0" argument(s): "The term 'Set-VMSwitch' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again."
![2020-07-29 Error in Virtual Switch config - 1](https://user-images.githubusercontent.com/62468943/88942262-531dc080-d28a-11ea-9ba1-edf05b5e5554.png)
![2020-07-29 Error in Virtual Switch config - 2](https://user-images.githubusercontent.com/62468943/88942273-5749de00-d28a-11ea-83ba-bdf6334af0b4.png)
I rebooted both nodes, but same error occurs, and I can't get past to enter "Clustering" step... Any idea ?
The build of WAC is Version 2007, 1.2.2007.18002 that I got from the download links at the beginning of the guide.

**Comment by mattmcspirit** [comment]
Hey - what language did you deploy with your nodes?

See here: https://github.com/Azure/AzureStackHCI-EvalGuide/issues/8

Your nodes are missing the Hyper-V PS, but I think there's some variation in the PS commands for en-us compared with other languages - i need to investigate more, but enabling "Hyper-V PowerShell" on your nodes would get you through this.

**Comment by laurenbo** [comment]
Hi Matt - my AZSHCI nodes are all en-US, with en-US culture, except for the keyboard, which is French.
What I found is that neither "RSAT-Hyper-V-Tools-Feature" nor "Microsoft-Hyper-V-Management-PowerShell" can be enabled due to the following error: One or several parent features are disabled so current feature can not be enabled.
    + CategoryInfo          : NotSpecified: (:) [Enable-WindowsOptionalFeature], COMException
    + FullyQualifiedErrorId : Microsoft.Dism.Commands.EnableWindowsOptionalFeatureCommand
    + PSComputerName        : AZSHCINODE01
I also found that this error do not appear anymore after the 1.5 step "Install Updates" (I had to run it twice) and the subsequent "Restart Servers" step. And the "Configure Adapter fail" for which I opened this issue is also solved. :-)
Hope this helps,
